openwisp_utils/admin.py

```diff
@@ -147,7 +144,7 @@ def receive_url(self, obj):
         )
         baseurl = self.receive_url_baseurl
         if not baseurl:
-            baseurl = '{0}://{1}'.format(self.request.scheme, self.request.get_host(),)
```

Comment:
there's many changes like this one which are not related to the goal of the PR, this makes history hard to follow.
Can you please separate these changes into a separate commit?

Reply:
@nemesisdesign I break down my commit in two, one for the changes and second for the formatting. Is it correct?

Reply:
> @nemesisdesign please review it.
> 
> Here I removed recursive checking on the subdirectories like docs as said. Also, **ignore_sphinx** is removed here. Although I don't rename **checkrst** to **checkreadme** as I was checking other rst files in root directory too. Like CHANGES.rst

@nemesisdesign I am changing the function to check only for readme instead because when I ran it on openwisp2-docs, it fails for index.rst file (as toctree is a sphinx directive and docutils doesn't recognize it).